code/game/objects/items/weapons/implants/implant.dm

```diff
@@ -12,16 +12,29 @@
 	var/allow_multiple = 0
 	var/uses = -1
 
+/obj/item/weapon/implant/New()
+	wires = new /datum/wires/simple_wire(src)
 
-/obj/item/weapon/implant/proc/trigger(emote, mob/source)
-	return
 
-/obj/item/weapon/implant/proc/activate()
+/obj/item/weapon/implant/proc/trigger(emote, mob/source)
 	return
 
 /obj/item/weapon/implant/ui_action_click()
 	activate("action_button")
 
+/obj/item/weapon/implant/attackby(obj/item/I, mob/user)
+	if(isassembly(I))
+		if(wires.attach_assembly(I))
+			user << "You attach [I] to [src]."
+			desc +=" It has [I] attached to it."

```

Comment:
Should probably override `examine()` to display `[I]` instead of changing the `desc`